Remotion/Validation/UnitTests/Merging/OrderPrecedenceValidationRuleCollectorMergerTest.cs

```diff
@@ -99,7 +99,7 @@ public void SetUp ()
       _removingPropertyValidationRuleCollector2.RegisterValidator(typeof(NotNullValidator), _validationRuleCollectorStub1.Object.GetType(), null);
 
       _removingPropertyValidationRuleCollector3 = RemovingPropertyValidationRuleCollector.Create(_firstNameExpression, typeof(CustomerValidationRuleCollector1));
-      _removingPropertyValidationRuleCollector3.RegisterValidator(typeof(NotNullValidator), typeof(string), null); //Unknown collector type!
+      _removingPropertyValidationRuleCollector3.RegisterValidator(typeof(NotNullValidator), _validationRuleCollectorStub1.Object.GetType(), null);
```

Comment:
This change is okay because the old code explicitly tested incorrect calls but there's no actual good reason to do this.